src/test/cto_multiple_pools/drd-log.supp

```diff
@@ -0,0 +1,9 @@
+{
+    <ut_log_suppress>
+    drd:ConflictingAccess
+    fun:*mempcpy
+    fun:_IO_file_xsputn@@GLIBC*
+    fun:fputs
+    ...
+    fun:ut_out
```

Comment:
Why is this file in test directory? Could we merge it with the one in src/test/?